agent/result_parser.py

```diff
@@ -108,4 +112,16 @@ def parse_results(results: Dict):
                         cvss_v3_vector=''
                     ),
                     technical_detail=technical_detail,
-                    risk_rating=_map_risk_rating(risk_rating_id, confidence_id))
+                    risk_rating=_map_risk_rating(risk_rating_id, confidence_id),
+                    vulnerability_location=vuln_mixin.VulnerabilityLocation(asset=domain_name.DomainName(name=host),
```

Comment:
This would depend on what is scanned, If the target is IP, we should report IP as the vulnerability location, otherwise it is domain name indead.